packages/product-react/src/Tabs/TabView.test.tsx

```diff
@@ -0,0 +1,77 @@
+import Enzyme, { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
this should be in a global `setupTests.js` file similar to https://github.com/WeConnect/wework-web-monorepo/blob/master/jest/setupTests.ts